internal/doculint/doculint.go

```diff
@@ -330,11 +330,11 @@ func validateGenDeclTypes(reporter nolint.Reporter, expr *ast.GenDecl) {
 // that if it is a variable block that the block itself has a comment, and each variable
 // within it also has a comment. If it is a standalone variable it ensures that it has a
 // comment associated with it.
-func validateGenDeclVariables(reporter nolint.Reporter, expr *ast.GenDecl) {
+func validateGenDeclVariables(report reporter.Reporter, expr *ast.GenDecl) {
```

Comment:
"Is it because you are an artifact of a reporter?"

"No, it's because I'm loud. Like, 'report of a rifle'."